transport.go

```diff
@@ -134,3 +135,19 @@ func (t *Transport) refreshToken() error {
 
 	return nil
 }
+
+func addAcceptHeader(headers http.Header) {
+	if headers.Get("Accept") == "" {
+		headers.Set("Accept", acceptHeader)
+		return
+	}
+
+	//Need to loop through all Accept headers incase there is more than one.
```

Comment:
Minor style issue. Since this is a normal comment for humans rather than for the compiler, it should have a space after `//`, like so:

```
// Need to ...
```

See https://dmitri.shuralyov.com/idiomatic-go#comments-for-humans-always-have-a-single-space-after-the-slashes for rationale.

Also, “incase” should be two words I think.

Reply:
@cpheps Did you see this comment?

Reply:
Missed the second one, just pushed a fix.

Reply:
What about the first part? I still see some comments that don't have a space after `//`.